### PR TITLE
✨ Enhancement/use sticky session

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?

This PR adds the sticky sessions to the webserver.

This means if a client connect with oSparc, Traefik will inject a cookie in the response, then recognize it on the next request from the same client and re-direct it to the very same instance of the webserver (in case they are scaled).

This is a pre-requisite to allow scaling of the webserver later. among others:
- handle the garbage collector (have only one instance do this - waiting for enhanced settings so that we have a garbage collector only service)
- add horizontal scaling using Redis or RabbitMQ

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -227,6 +227,7 @@ printf "$$rows" Portainer 'http://$(get_my_ip).nip.io:9000' admin adminadmin;\
 printf "$$rows" Redis 'http://$(get_my_ip).nip.io:18081';\
 printf "$$rows" 'Docker Registry' $${REGISTRY_URL} $${REGISTRY_USER} $${REGISTRY_PW};\
 printf "$$rows" "Dask Dashboard" "http://$(if $(IS_WSL2),$(get_my_ip),127.0.0.1).nip.io:8787";
+printf "$$rows" "Traefik Dashboard" "http://$(if $(IS_WSL2),$(get_my_ip),127.0.0.1).nip.io:8080/dashboard/";
```
Comment (pcrespov) [inline Makefile:230]: when i work remotely, I would like to have the ip, otherwise I have to be forwarding all ports :-)

**Review by pcrespov — APPROVED** [review]
minor comment on the ip change

**Review by GitHK — APPROVED** [review]
👍

File: Makefile (hunk 1 above)
Comment (GitHK) [reply Makefile:230]: I think we should only use the `$(get_my_ip)`, we already did this above.

Comment (pcrespov) [reply Makefile]: @sanderegg  so why we don't?

**Pull request merged**